accounts/abi/bind/bind_test.go

```diff
@@ -75,11 +97,34 @@ var bindTests = []struct {
 				t.Fatalf("binding (%v) nil or error (%v) not nil", b, nil)
 			}
 		`,
+		``,
+		`
+			if b, err := NewToken(common.Address{}, nil); b == nil || err != nil {
+				t.Fatalf("binding (%v) nil or error (%v) not nil", b, nil)
```

Comment:
This is currently failing: 

```
    bind_test.go:2204: test 1: failed to generate v2 binding: 77:39: expected ')', found ','
```

Reply:
This is because the output from abigen v2 is 
```go

	func (_Token *Token) PackConstructor(, owner common.Address ) ([]byte, error) {
		return _Token.abi.Pack("" , owner)
	}
```
and we are trying to format the output.

Reply:
fixed here [ea664d0](https://github.com/s1na/go-ethereum/pull/11/commits/ea664d081b3cf5ab3ba5fabf869ebc464339a7fe)